jupyter/Python/stac/stac-introduction.ipynb

```diff
@@ -579,6 +579,37 @@
     "print(rst.shape)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you wish you can use lat long coordinates and reproject them into the Albers Equal Area Projection, which is used in the BDC products:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pyproj import Transformer\n",
+    "from pyproj.crs import CRS\n",
+    "\n",
+    "inProj = CRS.from_epsg(4326)\n",
+    "outProj = CRS.from_user_input(src.crs)\n",
+    "transformer = Transformer.from_crs(inProj, outProj)\n",
+    "x1, y1 = -14.5980239872823, -47.40906064812335\n",
```

Comment:
It seems like `x1` and `y1` (or `x2` and `y2`) are inverted. It should be `x` as `longitude` and `y` as latitude. In this case, the parameter `always_xy=True` is required, according https://pyproj4.github.io/pyproj/stable/_modules/pyproj/transformer.html#Transformer.from_crs